lineage/Analyze.py

```diff
@@ -63,7 +63,10 @@ def run_Analyze_over(list_of_populations, num_states, parallel=True, **kwargs):
 
         prom_holder = []
         for idx, population in enumerate(list_of_populations):
-            prom_holder.append(exe.submit(Analyze, population, num_states[idx], fpi=list_of_fpi[idx], fT=list_of_fT[idx], fE=list_of_fE[idx]))
+            if isinstance(population, list):
```

Comment:
@Farnazmdi I was wrong—it will be hard to determine what you want to do here based on population. You'll want to bring back your option to `run_Analyze_over` for atonce fitting.

I can't get the build errors to happen. When running the parallel code, the underlying error doesn't get output, so it's hard to say what failed.

Reply:
You can take it from here.

Reply:
I will do that, thank you.

Reply:
Finally, check by putting `assert False` in the right places that you indeed are *only* getting atonce fitting in Fig. 9, and are not running it in another figure where it shouldn't be called.